projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/yes-no/read-yes-no-field.component.ts

```diff
@@ -4,7 +4,7 @@ import { YesNoService } from './yes-no.service';
 
 @Component({
   selector: 'ccd-read-yes-no-field',
-  template: `<span class="text-16">{{formattedValue}}</span>`
+  template: `<span class="text-16">{{caseField.label | rpxTranslate : null : formattedValue}}</span>`
```

Comment:
Do we need a null check something like this?
`caseField?.label | rpxTranslate : null`

Reply:
@udaydenduluri33  caseField null is handled in base component and passed as a input to this component. So I think null check is not need here. But I can add it as an extra check. Please, let me know